docs/reference/monitoring.txt

```diff
@@ -41,7 +41,10 @@ logging subscriber used internally by the driver:
     include Mongo::Loggable
 
     def started(event)
-      log_debug("#{prefix(event)} | STARTED | #{format_command(event.command)}")
+      # The default inspection of a command which is a BSON document gets
+      # truncated in the middle. To get the full rendering of the command, the
+      # ``to_json`` method can be called on the document.
+      log_debug("#{prefix(event)} | STARTED | #{format_command(event.command.to_json)}")
```

Comment:
I don't think this is quite correct:

```
irb(main):004:0> puts ({a:4}.to_json.inspect)
"{\"a\":4}"
=> nil
irb(main):005:0> puts ({a:4}.inspect)
{:a=>4}
=> nil
```
This code will output escaped json.

Perhaps
```
      log_debug("#{prefix(event)} | STARTED | #{event.command.to_json}")
```